cstore_fdw.c

```diff
@@ -62,7 +62,7 @@
 #if PG_VERSION_NUM >= 100000
 #define CALL_PREVIOUS_UTILITY(parseTree, queryString, context, paramListInfo, \
 							  destReceiver, completionTag) \
-	PREVIOUS_UTILITY(plannedStatement, queryString, context, paramListInfo, \
+	PREVIOUS_UTILITY(localPlannedStatement, queryString, context, paramListInfo, \
```

Comment:
we should not need this

Reply:
It is needed since this variable is not passed by the macro call. We have to point it to the modified copy of the PlannedStmt so that other ProcessUtility hooks do not see cstore tables.